docs/api/admin/tags-api.md

```diff
@@ -112,7 +112,26 @@ Used to fetch all types the server knows about. This endpoint is the one all adm
 }
 ```
 
-# Create a new tag type
+### Get a single tag type
+
+`GET: https://unleash.host.com/api/admin/tag-types/simple`
+
+Used to fetch details about a specific tag-type. This is mostly provded to make it easy to debug the API and should not be used by the client implementations.
+
+**Example response:**
+
+```json
+{
+  "version": 1,
+  "tagType": {
+    "name": "simple",
+    "description": "Some description",
+    "icon": "Some icon"
+  }
```

Comment:
should we also have a "createdAt" field?

Reply:
Yes, good point. I'll add it

Reply:
Do we want it for tags as well?

Reply:
I am unsure of the value, but it does not hurt to at least have the column in the dB, we can expose it later. 

It might be more valuable to have it on the coupling table (`feature_tag`).

Reply:
They're different cases really; when the tag was created, vs when a given feature was tagged.

Reply:
this is a good point. I think we can have both, I think it can be useful in some scenarios.